Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -114,6 +132,8 @@ public static class TestObjects {
             Weight = new Random().NextDouble(),
             Engine = new Engine { Cylinders = 4, ManufactureDate = DateTime.Parse("2009-10-10T13:15:21Z") }
         };
+        
+        public static Motorcycle harley = new Motorcycle("Harley", "Black", true, DateTime.MinValue);
```

Comment:
Should be readonly and variable name should be all caps - unless for some reason it's using a different convention here?